src/ts/components/seekbarcontroller.ts

```diff
@@ -0,0 +1,121 @@
+import { UIUtils } from '../uiutils';
+import { DOM } from '../dom';
+import { PlayerAPI } from 'bitmovin-player';
+import { VolumeController } from '../volumecontroller';
+
+export enum SeekBarType {
+    Vod,
+    Live,
+    Volume,
+}
+
+interface Range {
+    min: number;
+    max: number;
+}
+
+interface KeyStepIncrementsConfig {
+    leftRight: number;
+    upDown: number;
+}
+
+const changeRangeValue = (
```

Comment:
Methods like this one are usually called something with "coerce", e.g. `coerceValueIntoRange`

Reply:
Not sure why if you can explain? It just changes the value of the slider

Reply:
Because a coercion is what you do here. You force a value into a specific range.